sensor_module/sensor_module/src/src/hall_effect_sensor.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * hall_effect_sensor.c
+ *
+ * Created: 2019-11-25 13:05:48
+ *  Author: herap603
+ */
+
+#include "interrupt.h"
+
+#include "hall_effect_sensor.h"
+#include "sensor_data.h"
+
+/*
+The speed for each wheel is calculated as this constant divided by the counter
+value during the hall effect interrupt. This constant has been derived using
+speed = movement per interrupt / (counter value / count per second) where
+movement per interrupt = 0.25 dm. count per second is clock speed / prescaler
+i.e. 16 000 000 / 64 = 250 000. The constant is thus 250 000 * 0.25 = 62 500.
+This constant must be recalculated if any of the values used are changed.
+*/
+static const uint16_t counter_to_speed_constant = 62500;
```

Comment:
Constants should be written in UPPER_CASE preferably.

Reply:
Fixat.